docs/dg/learningBasics.md

```diff
@@ -358,15 +358,15 @@ Here are some small tasks for you to gain some basic knowledge of the code relat
 
   Add this to `c_summary.scss`.
 
-  ```
+  ```css
   .active-text {
     color: mui-color('green');
   }
   ```
 
   In `c-summary-charts.vue`, locate `summary-chart__title--name`, and add the following to its `v-bind:class` attribute map.
 
-  ```
+  ```vue
```

Comment:
I don't believe MarkBind supports the `vue` code block, perhaps we could either use another code block syntax highlighting, or remove it altogether.

Reply:
I see, I have removed the `vue` tag from the code, it should look better now!